### DISCUSSION TITLE
Styles Package?

### DISCUSSION THREAD

**Discussion by michael-hawker** [CommunityToolkit/Labs-Windows#581] [Ideas]
I can't find them at the moment, but over the years we have discussed whether or not we have ship loose styles that work with various system components.

@niels9001 has a couple he thinks could be useful to share:
- For NavigationView (to make it look like the Store pattern) 
- A subtle Button style
- [WinUI 3] Segmented Style for Selector Bar

We think we could add a `Styles` component/package which just contains these loose styles to make it easier for folks to consume and re-use. Especially as we don't have a good general package anymore for anything like this, grouping them all together in a Styles package seems to make sense.

Just opening this issue to track if there are any other prior or new examples of alternate styles, for example, maybe an alternate TabView style would fall in this bucket as well? https://github.com/microsoft/microsoft-ui-xaml/issues/7032

**Comment by ghost1372** [comment]
excellent
I know there is segmented control in toolkit But this was for a time when there was no selector bar control.
Therefore, having a segmented style for the selector bar is very useful

**Reply by michael-hawker** [reply]
Thanks @ghost1372, good call out. That's true the new SelectorBar only has the one style.

@niels9001 we'd had to figure out as part of this how we could separate out the one style that'd only be valid for WinUI 3... 🤔

**Reply by Arlodotexe** [reply]
> @niels9001 we'd had to figure out as part of this how we could separate out the one style that'd only be valid for WinUI 3... 🤔

Good call out. Compilation conditionals are only available in C#, but maybe we could include/exclude per-file (i.e. xaml, images, etc) in the csproj where MultiTarget and TFM information is available.

Or, if we wanted to make this more broadly reusable, we could make "per-file multitargets" an inbox feature of our tooling, using the file name to declare the compatible target(s).

**Comment by niels9001** [comment]
❤️         Niels Laute reacted to your message
